### PR TITLE
fix(orc8r): Updated helm to show orc8r version on NMS dashboard.

### LINKED ISSUE #13902: The Version of orchestrator is not showing .
Affected Component: Orchestrator, NMS
Affected Subcomponent: Orchestrator 

With updated Orchestrator, the Version of orchestrator is not showing .

### PR DESCRIPTION
**Pull request by tapasmishra**
## Summary

Fixes for #13902 

## Test Plan

Deploy the orc8r using the fixed helm and we should be able to see the version. 

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by ShubhamTatvamasi — APPROVED** [review]

**Comment by voisey** [comment]
What's the status here? It looks like a rebase might be needed to get the CI to run.

**Comment by voisey** [comment]
Hey @tapasmishra, could you please rebase this and then set the "ready2merge" label if you think it's good to go?

**Comment by maxhbr** [comment]
> Hey @tapasmishra, could you please rebase this and then set the "ready2merge" label if you think it's good to go?

Hey @tapasmishra, you never set "ready2merge" here, still not good to go?

File: .github/workflows/build_all.yml [hunk 1]
```diff
@@ -109,6 +109,7 @@ jobs:
           python ci-scripts/helm_repo_rotation.py
   agw-build:
     uses: ./.github/workflows/bazel.yml
+    secrets: inherit
```
Comment (maxhbr) [inline .github/workflows/build_all.yml]: this adds an independent fix to some completely unrelated PR. The title does not even mention this. Please keep such stuff separated.

**Review by maxhbr — CHANGES_REQUESTED** [review]

**Pull request closed without merging**